halo2_proofs/src/circuit/floor_planner/v1.rs

```diff
@@ -481,6 +481,14 @@ impl<'r, 'a, F: Field, CS: Assignment<F> + 'a> RegionLayouter<F> for V1Region<'r
         Ok(())
     }
 
+    fn name_column<'v>(
+        &'v mut self,
+        _annotation: &'v (dyn Fn() -> String + 'v),
+        _column: Column<Any>,
+    ) {
+        // Do nothing
+    }
```

Comment:
Because the only place where we annotate is in the MockProver. Hence, no Layouter is required to support this(as that will impact production performance).

Also, there is no usecase to debug with a real prover AFAIK.

Reply:
But no matter in which place, `Region::name_column` will always call `RegionLayouter::name_column` (`RegionShape` or `SingleChipLayouterRegion` or `V1Region`).

It makes sense to do nothing in `RegionShape`, but I can't see the difference between `SingleChipLayouterRegion` and `V1Region` (you can also try to update the `FloorPlanner` to `V1` and you will see the annotation disappear).

Reply:
I did it because use the `V1` for debugging is a bit weird. As takes more time and the constraint errors will be the same. 

Happy anyway to implement it for `V1`.